subjects/check_user_name/README.md

```diff
@@ -0,0 +1,64 @@
+## check_user_name
+
+### Instructions
+
+Sometimes it is more desirable to catch the failure of some parts of a program instead of just calling panic.
+
+For this exercise you will have to create a tool that manages users' access levels.
+
+You will have to create an `AccessLevel` enum which could be `Guest`, `Normal`, `Admin`.
+
+You will also have to create a `User` struct which has:
+
+- Fields:
+  - `name`: `String`
+  - `acessLevel`: `enum`
+- Associated functions:
+  - `new`: which initializes the struct.
+  - `send_name`: which takes only `self` as argument and returns an `Option<&str>` with `None` if the user is a guest or the `name` if the `AccessLevel` has any of the other options.
```

Comment:
I would highlight the word `Guest` to make it even clearer

Reply:
Thanks. Done